### ISSUE TITLE
Not able to login from webApp

### ISSUE THREAD

**Issue by Shankymahajan** [wireapp/wire-webapp#45]
I tried login in with my email ID . I did confirm my email , still I'm not able to confirm my account .

**Comment by bennycode** [comment]
Hi @Shankymahajan and thanks for filing this. Do you see an error in your browsers JavaScript console like [Chrome Dev Tools](https://developers.google.com/web/tools/chrome-devtools/)?

**Comment by Shankymahajan** [comment]
![rsz_screenshot_from_2016-07-31_224243](https://cloud.githubusercontent.com/assets/13821954/17278051/69e5ffba-5770-11e6-9161-9af15472f6e9.png)

Hi @bennyn I got this error:
 Failed to load resource: https://prod-nginz-https.wire.com/login?persist=false        the server responded with a status of 403 (Forbidden)

second point is as you can see in the screenshot I'm not able to login in as I need to confirm my account but I did tried it many times but was not able to login in .

**Comment by bennycode** [comment]
@Shankymahajan Can you tell us which email address you used to sign up? It's hard to read from your screenshot.

**Comment by Shankymahajan** [comment]
@bennyn sure shashankmahajan18@gmail.com

**Comment by bennycode** [comment]
@Shankymahajan thanks! Do you use Firefox to login?

**Comment by Shankymahajan** [comment]
I have tried 2 browsers for login - mozila ,chrome

**Comment by bennycode** [comment]
@Shankymahajan do you also use [Wire for desktops](https://wire.com/download)?

**Comment by bennycode** [comment]
@Shankymahajan Can you please try again to login on https://app.wire.com/?

**Comment by fdelapuente** [comment]
Hi,

I consistently can't login from a Private Browsing session in firefox on a Windows 7 machine.

I would like to confirm if it's actually not supported due to the device signature creation in a volatile session or something else?

In fact I wouldn't like anything to be stored at all in this computer, that's why I connect using a private session (not my personal computer)...

Thanks!

**Comment by gregor** [comment]
Wire does not work with Private Browsing on Firefox we needs access to IndexedDB. E2E-Encryption relies on the concept of devices. You can sign in with a non-permanent one (by leaving "Remember me" blank") when you sign in on Firefox outside a private browsing session. You will not have access to any of your history but receive new messages while you are logged in. Once you manually sign out from Wire on the browser we will clear the data stored in the IndexedDB.

**Comment by fdelapuente** [comment]
I just need to add that the issue title matches my issue, but I'm not getting the same error. In my case it says:  "Problems with the connection. Please try again."

**Comment by fdelapuente** [comment]
Thanks @gregor for the information about the behavior without "remember me"!

**Comment by gregor** [comment]
@fdelapuente We need to improve on our error messaging. "Problems with the connection. Please try again." is essentially the fallback if we do not have a more specific message, I guess we could catch the Firefox Private Browsing case more appropriately and link to the FAQ (https://support.wire.com/hc/en-us/articles/202960551), which could also be more explicit in explaining this limitation.

**Comment by fdelapuente** [comment]
Hi,

My father created an account some time ago with firefox on ubuntu 14.04. When he tried to login it said "Problems with the connection". Now he upgraded to ubuntu 16.04 and is running firefox 49.0 (the same version I have). He still gets the same messange when login in. I assisted him and confirmed that he resetted password and tried to login correctly with the same results. In other words he has never been able to login to his account.

When he tried to create a new account, it sais there's already an account with this e-mail (true).

What could be wrong?
Is it possible to delete his account in order to create a new one from scratch?

Thanks,
Felipe.

**Comment by fdelapuente** [comment]
It's working on chrome on the same device.

**Comment by bennycode** [comment]
@fdelapuente: Can you please open the JavaScript console from Firefox and report what shows up there? https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console

**Comment by fdelapuente** [comment]
Hi bennyn,

We did that today, and nothing rang a bell:

![wire_issue1](https://cloud.githubusercontent.com/assets/22240862/19403095/4bb915bc-923b-11e6-8fa4-f95b119c066a.png)

**Comment by gregor** [comment]
Closing this as there has not been any activity for months.

**Comment by Luckz** [comment]
Same message, should I make a new issue report?

My adblocker or similar is doing "GET https://web.localytics.com/v3/localytics.min.js net::ERR_BLOCKED_BY_CLIENT", beyond that console shows nothing.

**Comment by schlichtanders** [comment]
I have a similar problem, which is kind very annoying because I started to advertise wire a lot. If the web interface is broken, this is a show breaker unfortunately...

here my screenshot with crome developer tools

![wirewebloginproblems](https://cloud.githubusercontent.com/assets/4255480/25656821/1e6b6f98-2ffb-11e7-9629-e9ed3e819130.PNG)

**Comment by bennycode** [comment]
@schlichtanders Thank you very much for your report! Are you using any Chrome extensions?

**Comment by schlichtanders** [comment]
woow, in fact when running in inkognito mode, everything works.
however if if I disabled all extensions in normal mode, it does not work

**Comment by lipis** [comment]
If it works in incognito, then it's a bit strange. Maybe you should also clean the cache.

**Comment by bennycode** [comment]
@schlichtanders, what worries me is that it wants to load something called [MathJax](https://www.mathjax.org/) which is forbidden by our content security policy. So do you use a mathmatical extension for Chrome?

**Comment by schlichtanders** [comment]
it gets even stranger. Now after cleaning and restart, everything still disabled and third party cookies being activated, I get the following

![wirewebloginproblems2](https://cloud.githubusercontent.com/assets/4255480/25657918/7b1bff5a-3000-11e7-8fc2-4bdc8a5f71e1.PNG)
note that the remark that there are connection problems is now lost, however it still does not connect, but stays at this screen

[at your last comment: in fact I HAD a mathjax extension enabled which succesfully circumvents this security policy on many pages (I wanted to have beautiful mathjax rendering in chats - this was the purpose). Now the chrome extension board shows everything to be disabled]

**Comment by schlichtanders** [comment]
unfortunately the very similar now happens in Inkognito mode...
and also the Windows app seems to be frozen now.

This seems to be a real bug in that I guess I just logged in too many times, however wire is not telling me anything on the UI level, however I can only see hints at the console. There should be some information, most crucially how long the account is blocked...

I have created a new issue for this https://github.com/wireapp/wire/issues/37, however the other issue still is there - I am unable to login, in fact the trials so far made even worse

**Comment by bennycode** [comment]
Thanks for creating the ticket. We have to do something about the "logged in too often" issue. There is only a programmatic way to resolve this issue:

```javascript
UserAPI.prototype.removeCookies = function(labels) {
  let self = this;

  return popsicle.request({
    method: 'POST',
    url: `https://prod-nginz-https.wire.com/cookies/remove`,
    body: {
      email: self.user.email,
      password: self.user.password,
      labels,
    },
    headers: {
      'Content-Type': 'application/json; charset=utf-8',
    },
  }).use([status(), popsicle.plugins.parse('json')]);
};
```

**Comment by schlichtanders** [comment]
thank you very much for further helping out

I tried to execute your javascript snippet in the console of app.wire.com
however I got ``Uncaught ReferenceError: UserAPI is not defined``

Is there a timelimit after which I am no longer blocked? Can you unblock me?

**Comment by Luckz** [comment]
Today it works for me - both on Firefox and Chrome.

**Comment by schlichtanders** [comment]
[the block seems to be gone after one day, that is at least something]

the chrome extension which seem to don't work well with wire is ``tex-all-the-things`` https://chrome.google.com/webstore/detail/tex-all-the-things/cbimabofgmfdkicghcadidpemeenbffn?hl=en

this is unfortunate as texifying things is a really useful feature to have.
Anyone knowing what to do next with this incompatibility issue? (it is not urgent, but still something to be solved ideally)
